api/server/router/container/container_routes.go

```diff
@@ -386,10 +385,6 @@ func (s *containerRouter) deleteContainers(ctx context.Context, w http.ResponseW
 	}
 
 	if err := s.backend.ContainerRm(name, config); err != nil {
-		// Force a 404 for the empty string
```

Comment:
Just realized this code would produce a 500 error and not a 404 anyhow. I guess it's fine to remove then. It'll return a 400 error instead.